HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -110,6 +109,193 @@ def test_ducts
     _test_default_duct_values(hpxml, expected_supply_locations, expected_return_locations, expected_supply_areas, expected_return_areas, expected_n_return_registers)
   end
 
+  def test_pv
+    hpxml_name = 'base-pv.xml'
+    hpxml = HPXML.new(hpxml_path: File.join(@root_path, 'workflow', 'sample_files', hpxml_name))
+    XMLHelper.write_file(hpxml.to_oga, @tmp_hpxml_path)
+    model, hpxml = _test_measure(@args_hash)
+    expected_interver_efficiency = [0.96, 0.96]
+    expected_system_loss_frac = [0.14, 0.14]
+    _test_default_pv(hpxml, expected_interver_efficiency, expected_system_loss_frac)
+
+    default_hpxml('base-pv.xml')
+    model, hpxml = _test_measure(@args_hash)
+    expected_interver_efficiency = [0.96, 0.96]
+    expected_system_loss_frac = [0.14, 0.14]
+    _test_default_pv(hpxml, expected_interver_efficiency, expected_system_loss_frac)
+
+    hpxml = default_hpxml('base-pv.xml')
+    hpxml.pv_systems.each do |pv|
+      pv.year_modules_manufactured = 2010
+    end
+    XMLHelper.write_file(hpxml.to_oga, @tmp_hpxml_path)
+    model, hpxml = _test_measure(@args_hash)
+    expected_interver_efficiency = [0.96, 0.96]
+    expected_system_loss_frac = [0.182, 0.182]
+    _test_default_pv(hpxml, expected_interver_efficiency, expected_system_loss_frac)
+  end
+
+  def test_conditioned_building_volume
+    hpxml_name = 'base.xml'
+    hpxml = HPXML.new(hpxml_path: File.join(@root_path, 'workflow', 'sample_files', hpxml_name))
+    XMLHelper.write_file(hpxml.to_oga, @tmp_hpxml_path)
+    model, hpxml = _test_measure(@args_hash)
+    _test_default_conditioned_building_volume(hpxml, 21600)
+
+    default_hpxml('base.xml')
+    model, hpxml = _test_measure(@args_hash)
+    _test_default_conditioned_building_volume(hpxml, 27000)
+  end
+
+  def test_appliances
+    hpxml_name = 'base.xml'
+    hpxml = HPXML.new(hpxml_path: File.join(@root_path, 'workflow', 'sample_files', hpxml_name))
+    XMLHelper.write_file(hpxml.to_oga, @tmp_hpxml_path)
+    model, hpxml = _test_measure(@args_hash)
+    _test_default_clothes_washer(hpxml, [HPXML::LocationLivingSpace, 1.21, 380.0, 0.12, 1.09, 27.0, 3.2, 6.0, 1.0])
+    _test_default_clothes_dryer(hpxml, [HPXML::LocationLivingSpace, HPXML::ClothesDryerControlTypeTimer, 3.73, 1.0])
+    _test_default_dish_washer(hpxml, [HPXML::LocationLivingSpace, 307.0, 0.12, 1.09, 22.32, 4.0, 12, 1.0])
+    _test_default_refrigerator(hpxml, [HPXML::LocationLivingSpace, 650.0, 1.0])
+    _test_default_cooking_range(hpxml, [HPXML::LocationLivingSpace, false, 1.0])
+    _test_default_oven(hpxml, false)
+
+    hpxml = default_hpxml('base.xml')
+    hpxml.header.eri_calculation_version = '2014ADEGL'
```

Comment:
I don't understand, why do you need to do this?

Reply:
Oh, are you specifically trying to test appliances from this ERI version? If you had a comment saying what you were trying to test, I would have understood. Note that the appliances actually changed in 2019 Addendum A, so you can use `hpxml.header.eri_calculation_version = '2019'` and make the comment "# Test appliances before 2019 Addendum A" or whatever.